### PR TITLE
localStorage functions

### PR DESCRIPTION
**Pull request by mgwc**
Does it make sense to clear the localStorage in startGame()?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ import Start from './components/Start';
...
 import startImage from './images/start_screen.png';
+import { clearChoicesRecord } from "./utils/utils";
 
 function App() {
   const [isInStart, setIsInStart] = useState(true);
   const [questionsSeen, setQuestionsSeen] = useState(-1);
   const [isInInfo, setIsInInfo] = useState(false);
 
   const startGame = () => {
+    clearChoicesRecord()
```
Comment (meganweijiang) [inline src/App.js:18]: nice!

Comment (meganweijiang) [reply src/App.js:18]: we'll probably want a separate ticket for badges won to be stored in local storage and that wouldn't be cleared on game start

**Review by meganweijiang — APPROVED** [review]
LGTM!

**Review by chethana-gopinath — APPROVED** [review]
💯

File: src/App.js (hunk 1 above)
Comment (mgwc) [reply src/App.js:18]: Good idea, I can circle back to that

**Pull request merged**